gulpfile.ts

```diff
@@ -31,4 +31,4 @@ require('./scripts/gulp/tasks/test-circulars');
 require('./scripts/gulp/tasks/test-dependencies');
 
 // global tasks
-task('build', parallel('bundle:all-packages', 'build:docs'));
+task('build', parallel('build:docs'));
```

Comment:
Next step will be probably to remove `build` task, too

Reply:
I can remove it as part of this PR, I just wasn't sure if `build:docs` is still needed separately from `yarn build` or not. 

I also noticed `build:docs:json` was an explicit step in CircleCI pipeline when auditing pipeline steps. However, it doesn't seem to be needed in ADO pipeline, so I'm guessing that's covered by `yarn build`.

Reply:
>I also noticed build:docs:json was an explicit step in CircleCI pipeline when auditing pipeline steps. However, it doesn't seem to be needed in ADO pipeline, so I'm guessing that's covered by yarn build.

For ADO it's not needed indeed